src/lib/operators/get_table.cpp

```diff
@@ -1,27 +1,63 @@
 #include "get_table.hpp"
 
+#include <algorithm>
 #include <memory>
+#include <sstream>
 #include <string>
+#include <unordered_set>
 #include <vector>
 
 #include "storage/storage_manager.hpp"
+#include "types.hpp"
 
 namespace opossum {
 
-GetTable::GetTable(const std::string& name) : _name(name) {}
+GetTable::GetTable(const std::string& name) : _name(name), _excluded_chunks() {}
+
+GetTable::GetTable(const std::string& name, std::vector<ChunkID> excluded_chunks)
```

Comment:
pass per const ref (preferred by me) or move into target variable (preferred by others)

Reply:
i first did it that way, the problem is that `StoredTableNode` returns the excluded chunks as a const reference. which means we have to copy the values anyway.

Reply:
but now you're copying twice, once into the param, and once into the member variable